aggregate-snapshot/aggregate-snapshot-repository/src/test/java/uk/gov/justice/services/eventsourcing/jdbc/snapshot/SnapshotJdbcRepositoryJdbcIT.java

```diff
@@ -222,14 +237,27 @@ private void removeAllSnapshots() throws Exception  {
         }
     }
 
+    private Timestamp findSnapshotCreatedAt(UUID streamId, Long versionId) throws Exception {
+        try (final Connection connection = eventStoreDataSourceProvider.getDefaultDataSource().getConnection();
+             final PreparedStatement preparedStatement = connection.prepareStatement(FIND_CREATED_TIME_BY_VERSION_ID)) {
+             preparedStatement.setObject(1, streamId);
+             preparedStatement.setLong(2, versionId);
+             try(final ResultSet rs = preparedStatement.executeQuery()) {
+                 rs.next();
+                 return rs.getTimestamp("created_at");
```

Comment:
Do we need to check if(rs.next()) before getting the ts?
just in case nothing found matching criteria